package.json

```diff
@@ -3,9 +3,9 @@
   "version": "1.2.35-1",
   "private": true,
   "scripts": {
-    "android": "react-native run-android",
+    "android": "adb uninstall me.rainbow || true && react-native run-android",
```

Comment:
why this? It should override old build

Reply:
It doesn't when you switch from release mode to develop or viceversa and it's very annoying cause you have to manually delete app and re-run the entire command again